### PR TITLE
Cleaned Up frmAttendedMode.cs, frmCommandEditor.cs, frmScheduleManagement.cs

### PR DESCRIPTION
**Pull request by ob-ab**
Cleaned Up following form files under UI/Forms:
- frmAttendedMode.cs
- frmCommandEditor.cs
- frmScheduleManagement.cs

Updated references in dependent files

### CODE REVIEW

**Review by openbots-ff — DISMISSED** [review]
Looks good, just leaving a note that we should merge frmScriptBuilder first

**Review by melugi — DISMISSED** [review]

File: taskt/UI/Forms/frmScheduleManagement.cs [hunk 1]
```diff
@@ -39,16 +36,16 @@ public frmScheduleManagement()
         private void frmScheduleManagement_Load(object sender, EventArgs e)
         {
             //get path to executing assembly
-            txtAppPath.Text = System.Reflection.Assembly.GetEntryAssembly().Location;
+            txtAppPath.Text = Assembly.GetEntryAssembly().Location;
 
             //get path to scripts folder
-            rpaScriptsFolder = Folders.GetFolder(Core.IO.Folders.FolderType.ScriptsFolder);
+            _rpaScriptsFolder = Folders.GetFolder(Core.IO.Folders.FolderType.ScriptsFolder);
```
Comment (CameronHerwig) [inline taskt/UI/Forms/frmScheduleManagement.cs]: Might be able to reduce this namespace to Folder.FolderType.ScriptFolder

File: taskt/Core/Automation/Commands/Loop Commands/BeginMultiLoopCommand.cs [hunk 2]
```diff
@@ -202,13 +202,13 @@ private void CreateLoopCondition(object sender, EventArgs e)
...
+            editor.SelectedCommand = new BeginLoopCommand();
             var res = editor.ShowDialog();
 
             if (res == DialogResult.OK)
             {
                 //get data
-                var configuredCommand = editor.selectedCommand as BeginLoopCommand;
+                var configuredCommand = editor.SelectedCommand as BeginLoopCommand;
                 var displayText = configuredCommand.GetDisplayValue();
                 var serializedData = Newtonsoft.Json.JsonConvert.SerializeObject(configuredCommand);
```
Comment (CameronHerwig) [inline taskt/Core/Automation/Commands/Loop Commands/BeginMultiLoopCommand.cs:213]: Could likely be reduced to JsonConvert.SerializeObject

File: taskt/Core/Automation/Commands/Loop Commands/BeginMultiLoopCommand.cs [hunk 3]
```diff
@@ -164,15 +164,15 @@ private void LoopConditionHelper_CellContentClick(object sender, DataGridViewCel
...
+                    editor.OriginalCommand = loopCommand;
+                    editor.CreationModeInstance = frmCommandEditor.CreationMode.Edit;
+                    editor.ScriptVariables = ScriptVariables;
 
                     if (editor.ShowDialog() == DialogResult.OK)
                     {
-                        var editedCommand = editor.editingCommand as BeginLoopCommand;
+                        var editedCommand = editor.EditingCommand as BeginLoopCommand;
                         var displayText = editedCommand.GetDisplayValue();
                         var serializedData = Newtonsoft.Json.JsonConvert.SerializeObject(editedCommand);
```
Comment (CameronHerwig) [inline taskt/Core/Automation/Commands/Loop Commands/BeginMultiLoopCommand.cs:177]: Could likely be reduced to JsonConvert.SerializeObject

File: taskt/Core/Automation/Commands/Loop Commands/BeginMultiLoopCommand.cs [hunk 4]
```diff
@@ -202,13 +202,13 @@ private void CreateLoopCondition(object sender, EventArgs e)
             var automationCommands = taskt.UI.CustomControls.CommandControls.GenerateCommandsandControls().Where(f => f.Command is BeginLoopCommand).ToList();
```
Comment (CameronHerwig) [inline taskt/Core/Automation/Commands/Loop Commands/BeginMultiLoopCommand.cs:202]: Could likely be reduced to CommandControls.GenerateCommandsandControls

File: taskt/Core/Automation/Commands/Loop Commands/BeginMultiLoopCommand.cs [hunk 5]
```diff
@@ -164,15 +164,15 @@ private void LoopConditionHelper_CellContentClick(object sender, DataGridViewCel
 
                     var automationCommands = taskt.UI.CustomControls.CommandControls.GenerateCommandsandControls().Where(f => f.Command is BeginLoopCommand).ToList();
```
Comment (CameronHerwig) [inline taskt/Core/Automation/Commands/Loop Commands/BeginMultiLoopCommand.cs:165]: Could likely be reduced to CommandControls.GenerateCommandsandControls

File: taskt/Core/Automation/Commands/If Commands/BeginMutliIfCommand.cs [hunk 6]
```diff
@@ -215,13 +215,13 @@ private void CreateIfCondition(object sender, EventArgs e)
...
+            editor.SelectedCommand = new BeginIfCommand();
             var res = editor.ShowDialog();
 
             if (res == DialogResult.OK)
             {
                 //get data
-                var configuredCommand = editor.selectedCommand as BeginIfCommand;
+                var configuredCommand = editor.SelectedCommand as BeginIfCommand;
                 var displayText = configuredCommand.GetDisplayValue();
                 var serializedData = Newtonsoft.Json.JsonConvert.SerializeObject(configuredCommand);
```
Comment (CameronHerwig) [inline taskt/Core/Automation/Commands/If Commands/BeginMutliIfCommand.cs:226]: Could Likely be reduced to JsonConvert.SerializeObject

File: taskt/Core/Automation/Commands/If Commands/BeginMutliIfCommand.cs [hunk 7]
```diff
@@ -215,13 +215,13 @@ private void CreateIfCondition(object sender, EventArgs e)
             var automationCommands = taskt.UI.CustomControls.CommandControls.GenerateCommandsandControls().Where(f => f.Command is BeginIfCommand).ToList();
```
Comment (CameronHerwig) [inline taskt/Core/Automation/Commands/If Commands/BeginMutliIfCommand.cs:215]: Could likely be reduced to CommandControls.GenerateCommandsandControls

File: taskt/Core/Automation/Commands/If Commands/BeginMutliIfCommand.cs [hunk 8]
```diff
@@ -177,15 +177,15 @@ private void IfConditionHelper_CellContentClick(object sender, DataGridViewCellE
...
+                    editor.OriginalCommand = ifCommand;
+                    editor.CreationModeInstance = frmCommandEditor.CreationMode.Edit;
+                    editor.ScriptVariables = ScriptVariables;
 
                     if (editor.ShowDialog() == DialogResult.OK)
                     {
-                        var editedCommand = editor.editingCommand as BeginIfCommand;
+                        var editedCommand = editor.EditingCommand as BeginIfCommand;
                         var displayText = editedCommand.GetDisplayValue();
                         var serializedData = Newtonsoft.Json.JsonConvert.SerializeObject(editedCommand);
```
Comment (CameronHerwig) [inline taskt/Core/Automation/Commands/If Commands/BeginMutliIfCommand.cs:190]: Could likely be reduced to JsonConvert.SerializeObject

File: taskt/Core/Automation/Commands/If Commands/BeginMutliIfCommand.cs [hunk 9]
```diff
@@ -177,15 +177,15 @@ private void IfConditionHelper_CellContentClick(object sender, DataGridViewCellE
 
                     var automationCommands = taskt.UI.CustomControls.CommandControls.GenerateCommandsandControls().Where(f => f.Command is BeginIfCommand).ToList();
```
Comment (CameronHerwig) [inline taskt/Core/Automation/Commands/If Commands/BeginMutliIfCommand.cs:178]: Could likely be reduced to CommandControls.GenerateCommandsandControls

**Review by CameronHerwig — COMMENTED** [review]
Namespaces can most likely be shortened in a few placed.

File: taskt/Core/Automation/Commands/Loop Commands/BeginMultiLoopCommand.cs (hunk 2 above)
Comment (ob-ab) [reply taskt/Core/Automation/Commands/Loop Commands/BeginMultiLoopCommand.cs:213]: As you know, we are not cleaning up Command Files right now.
It's just updating variable names in dependent files.

File: taskt/Core/Automation/Commands/Loop Commands/BeginMultiLoopCommand.cs (hunk 3 above)
Comment (ob-ab) [reply taskt/Core/Automation/Commands/Loop Commands/BeginMultiLoopCommand.cs:177]: Same is the case here that it's just updating variable names/references in this Command File.

File: taskt/Core/Automation/Commands/Loop Commands/BeginMultiLoopCommand.cs (hunk 4 above)
Comment (ob-ab) [reply taskt/Core/Automation/Commands/Loop Commands/BeginMultiLoopCommand.cs:202]: Same is the case here that it's just updating variable names/references in this Command File.

File: taskt/Core/Automation/Commands/Loop Commands/BeginMultiLoopCommand.cs (hunk 5 above)
Comment (ob-ab) [reply taskt/Core/Automation/Commands/Loop Commands/BeginMultiLoopCommand.cs:165]: Same is the case here that it's just updating variable names/references in this Command File.

File: taskt/Core/Automation/Commands/If Commands/BeginMutliIfCommand.cs (hunk 6 above)
Comment (ob-ab) [reply taskt/Core/Automation/Commands/If Commands/BeginMutliIfCommand.cs:226]: Same is the case here that it's just updating variable names/references in this Command File.

File: taskt/Core/Automation/Commands/If Commands/BeginMutliIfCommand.cs (hunk 7 above)
Comment (ob-ab) [reply taskt/Core/Automation/Commands/If Commands/BeginMutliIfCommand.cs:215]: Same is the case here that it's just updating variable names/references in this Command File.

File: taskt/Core/Automation/Commands/If Commands/BeginMutliIfCommand.cs (hunk 8 above)
Comment (ob-ab) [reply taskt/Core/Automation/Commands/If Commands/BeginMutliIfCommand.cs:190]: Same is the case here that it's just updating variable names/references in this Command File.

File: taskt/Core/Automation/Commands/If Commands/BeginMutliIfCommand.cs (hunk 9 above)
Comment (ob-ab) [reply taskt/Core/Automation/Commands/If Commands/BeginMutliIfCommand.cs:178]: Same is the case here that it's just updating variable names/references in this Command File.

File: taskt/UI/Forms/frmScheduleManagement.cs (hunk 1 above)
Comment (ob-ab) [reply taskt/UI/Forms/frmScheduleManagement.cs]: done

**Review by CameronHerwig — APPROVED** [review]

**Review by melugi — APPROVED** [review]

**Pull request merged**